coderd/database/migrations/000037_jwt_licenses.up.sql

```diff
@@ -0,0 +1,11 @@
+-- No valid licenses should exist, but to be sure, drop all rows
+DELETE FROM licenses;
+ALTER TABLE licenses DROP COLUMN license;
+ALTER TABLE licenses RENAME COLUMN created_at to uploaded_at;
+ALTER TABLE licenses ADD COLUMN jwt text NOT NULL;
```

Comment:
In `sqlc.go` you can add a rename for this column so the output is `JWT` instead of `Jwt`.